source/features/vertical-front-matter.tsx

```diff
@@ -24,7 +24,7 @@ function init(): void | false {
 		return false;
 	}
 
-	const values = [...rows[0].children];
+	const values = [...rows[0].children as HTMLCollectionOf<HTMLElement>];
```

Comment:
We don't need this array. `{values[index]}` below can be changed to `{rows[0].children[index]}`

Reply:
I tried this, I did not fully understand what I was doing here.